packages/design-system/src/styles/styles2.scss

```diff
@@ -0,0 +1 @@
+@import "styles"
```

Comment:
Give file a better name, maybe `stylesWrapper.scss`  and explain it's needed so Vite does not complain the file is already loaded.